tests/lora/test_lora_layers_flux.py

```diff
@@ -478,21 +463,16 @@ def test_lora_expanding_shape_with_normal_lora_raises_error(self):
             "transformer.x_embedder.lora_B.weight": normal_lora_B.weight,
         }
 
-        # The first lora expanded the input features of x_embedder. Here, we are trying to load a lora with the correct
```

Comment:
So this error is now removed because expanding lora state dicts is now supported

Reply:
Exactly!